matter_server/server/__main__.py

```diff
@@ -95,6 +104,7 @@ def main() -> None:
         int(args.vendorid),
         int(args.fabricid),
         int(args.port),
+        args.listen_address,
```

Comment:
you are sending a None or single string value here while the typing of "listen_addresses" is set to a list of strings or None.

Reply:
nvm, I overlooked the append.